packages/node-firebird-driver/src/test/tests.ts

```diff
@@ -45,6 +46,14 @@ export function runCommonTests(client: Client) {
 			`${d.getUTCHours()}:${d.getUTCMinutes()}:${d.getUTCSeconds()}.${d.getUTCMilliseconds()} ${zd.timeZone}'`;
 	}
 
+	async function isFB4OrGreater(attachment: Attachment) {
```

Comment:
Please take this thing out of this PR.

It's not completely correct in its usage and has nothing directly to do with the topic.

Reply:
there are two places in tests which don't pass with FB3. 
one is INSERT INTO ... RETURNING *
and another NUMERIC(.., 20)

if you are ok with tests don't run under FB3, I will remove this check. I know that the better way is to get version of the server instead of ODS major version but how to obtain it?

Reply:
But in the way you do, many valid tests for FB 3 will not run. So lets do it correctly in separate patch.

This code will also need adjustments for FB 5.